mux/src/pane.rs

```diff
@@ -316,6 +528,10 @@ pub trait Pane: Downcast + Send + Sync {
     fn exit_behavior(&self) -> Option<ExitBehavior> {
         None
     }
+
+    fn save(&self) -> Option<SavedPaneState> {
```

Comment:
naming: match whatever naming we come up with for the mux level save function as discussed above

But, conceptually speaking the command resolution and replay needs some additional consideration that I'll break out into a separate non-code comment.